wherehows-web/app/utils/api/datasets/highlight-results.ts

```diff
@@ -0,0 +1,55 @@
+import { IDataset } from 'wherehows-web/typings/api/datasets/dataset';
+
```

Comment:
Maybe i'm missing it, but does this actually highlight the keywords?

Reply:
It used to, but I guess the naming is kind of misleading now.